data/sd/legislature/Linda-Duba-f46d0945-680c-40a2-b65d-868aae2b4ada.yml

```diff
@@ -0,0 +1,25 @@
+id: ocd-person/f46d0945-680c-40a2-b65d-868aae2b4ada
+name: Linda Duba
+given_name: Linda
+family_name: Duba
+email: Linda.Duba@sdlegislature.gov
+image: https://lawmakerdocuments.blob.core.usgovcloudapi.net/photos/https://lawmakerdocuments.blob.core.usgovcloudapi.net/photos/rt
+  duba, linda-4750.jpg
+party:
+- name: Democratic
```

Comment:
@sroomf i think this file should be deleted.

Reply:
Oh why though? Linda's representing district 15 - https://ballotpedia.org/Linda_Duba